### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by HuaTang92**
update readme based on https://github.com/artem-zinnatullin/AutoJackson/pull/10

### CODE REVIEW

**Review by artem-zinnatullin — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -16,12 +16,12 @@ It is **serializable** to JSON and **deserializable** from JSON, it's immutable
 // Just use @AutoParcel annotations if you need AutoParcel
 
 @AutoValue
-@JsonDeserialize(builder = AutoValue_Tweet.Builder.class)
+@JsonDeserialize(builder = Tweet.Builder.class)
```
Comment (artem-zinnatullin) [inline README.md]: Wait, `@JsonSerialize(as = Tweet.class)`

**Review by artem-zinnatullin — CHANGES_REQUESTED** [review]

**Comment by artem-zinnatullin** [comment]
looks good, but comment :)

File: README.md (hunk 1 above)
Comment (HuaTang92) [reply README.md]: Sure, it should have both serializer and deserializer.

**Review by artem-zinnatullin — APPROVED** [review]

**Comment by artem-zinnatullin** [comment]
Thanks!

**Pull request merged**